### PR TITLE
Jormungandr: those functions were called twice

### PR DESCRIPTION
**Pull request by antoine-de**

### CODE REVIEW

File: source/jormungandr/jormungandr/interfaces/v1/Journeys.py [hunk 1]
```diff
@@ -580,8 +580,6 @@ def __init__(self):
             parser_post.args[index].location = "json"
 
     @add_debug_info()
-    @clean_links()
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:583]: You can do the same for the post method I think

Comment (l-vincent-l) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:583]: Btw, it should be an OPTIONS method.

Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:583]: why ?

Comment (antoine-de) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:583]: remove in post done

Comment (l-vincent-l) [inline source/jormungandr/jormungandr/interfaces/v1/Journeys.py:583]: If you wanna be restful, post should create a resource.
Actually, OPTIONS is not in restful defintions...
So...

The good restful way to do it is to create requests resources, and then call it

**Pull request merged**